README.md

```diff
@@ -837,13 +1017,24 @@ You can reach me on [Twitter @ileriayooo](https://twitter.com/Ileriayooo)
 | Selenium        | ![Selenium](https://img.shields.io/badge/-selenium-%43B02A?style=for-the-badge&logo=selenium&logoColor=white)                       | `![Selenium](https://img.shields.io/badge/-selenium-%43B02A?style=for-the-badge&logo=selenium&logoColor=white)`                       |
 | Testing Library | ![Testing-Library](https://img.shields.io/badge/-TestingLibrary-%23E33332?style=for-the-badge&logo=testing-library&logoColor=white) | `![Testing-Library](https://img.shields.io/badge/-TestingLibrary-%23E33332?style=for-the-badge&logo=testing-library&logoColor=white)` |
 
+[(Back to top)](#table-of-contents)
+
+### Tor Projects
+
+| Name        | Badge                                                                                                                  | Markdown                                                                                                                 |
+|-------------|------------------------------------------------------------------------------------------------------------------------|--------------------------------------------------------------------------------------------------------------------------|
+| Tor         | ![Tor](https://img.shields.io/badge/Tor-7D4698?style=for-the-badge&logo=Tor-Browser&logoColor=white)                   | `![Tor](https://img.shields.io/badge/Tor-7D4698?style=for-the-badge&logo=Tor-Browser&logoColor=white)`                   |
+| Tor Browser | ![Tor Browser](https://img.shields.io/badge/Tor_Browser-%237D4698?style=for-the-badge&logo=TorBrowser&logoColor=white) | `![Tor Browser](https://img.shields.io/badge/Tor_Browser-%237D4698?style=for-the-badge&logo=TorBrowser&logoColor=white)` |
```

Comment:
I think we should have this under the Browser category.

Favor general (widely known) categories rather than specific (sometimes unknown) categories, so that it is easier to find. For instance, when a person thinks of Tor Browser, they'd want to find it where other browsers are listed.

Similarly, we have several Google products (Play, Drive, Chrome, etc), however, they are not listed under a category for Google.

Reply:
Hi! This is an excellent slice of information. Would be great if this could be refactored and added to the contribution guide. 

Let me send in a PR. @Ileriayo